frontend/src/components/modalContent/AddIncidentModal.js

```diff
@@ -156,8 +158,9 @@ const StepForms = (
         incidentStatusCodeOptions={props.incidentStatusCodeOptions}
         hasFatalities={props.addIncidentFormValues.number_of_fatalities > 0 || false}
         hasFollowUp={props.addIncidentFormValues.followup_inspection || false}
+        determinationTypeCode={props.addIncidentFormValues.determination_type_code}
```

Comment:
Was this prop missing?

Reply:
I added it in order to conditionally show a warning message about an uncommon use of the incident form, as per UX feedback.